src/Components/BottomBar.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import {
+  MdThumbUp,
+  MdHistory,
+  MdWatchLater,
+  MdHome,
+  MdPlaylistPlay
+} from "react-icons/md";
+import { Link } from "react-router-dom";
+export function BottomBar() {
+  return (
+    <div className="mobile-mode">
+      <div
+        style={{ justifyContent: "space-evenly" }}
+        className="bottom-bar flex"
+      >
+        <Link className="link-btn" to="/">
+          <MdHome size="1.5rem" />
```

Comment:
If all icons are of same size, then you can store this value as variable and assign this variable to size attribute.

Reply:
I have created a variable named iconSize in utils.

Reply:
looks good.